dumper/dataWriter.go

```diff
@@ -191,7 +193,7 @@ func substitutePrimaryKey(table schemareader.Table, row []sqlUtil.RowDataStructu
 	return rowResult
 }
 
-func substituteForeignKey(db *sql.DB, table schemareader.Table, tables map[string]schemareader.Table, row []sqlUtil.RowDataStructure) []sqlUtil.RowDataStructure {
+func SubstituteForeignKey(db *sql.DB, table schemareader.Table, tables map[string]schemareader.Table, row []sqlUtil.RowDataStructure) []sqlUtil.RowDataStructure {
```

Comment:
why does this method need to be exposed outside the dumper module?

Reply:
@rjmateus because the [PostOrderCallback](https://github.com/uyuni-project/inter-server-sync/pull/22/commits/ce3b8ba02a822eb229ff4f8db97e963a2ee1b4d6#diff-50c9b9274074412cc6ea92b60e704bbd606ff73161a2b1922489c5ce57b0d4e8R102) now resides in a different package and needs this method to be public. Please tell me if we could improve the separation of concerns here. Otherwise I would leave it for the refactoring.